autoscale-dockerswarm-container/pom.xml

```diff
@@ -40,43 +40,52 @@
         <dependency>
             <groupId>com.github.cafapi</groupId>
             <artifactId>caf-api</artifactId>
+            <scope>runtime</scope>
         </dependency>
```

Comment:
I still think `caf-api` should have been removed.  All that should be here is the `autoscale-core` and the various drop-in modules (i.e. choice of codec, choice of config etc.).

Reply:
caf-api is removed.